### ISSUE TITLE
Open link in room description in browser

### ISSUE THREAD

**Issue by obstschale** [RocketChat/Rocket.Chat.Electron#2592]
### Search before asking

- [X] I had searched in the [issues](https://github.com/RocketChat/Rocket.Chat.Electron/issues?q=is%3Aissue) and found no similar issues.

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

### Operating System Version

macOS 13.1

### It happens on the web browser too?

No, it just happens on the Desktop app

### Rocket.Chat Desktop App Version

3.8.16

### Rocket.Chat Server Version

5.4

### Describe the bug

Links in room descriptions open in the client not in your browser.

### How to Reproduce

1. Add a text as room description. The text includes a markdown link:

```md
My room description. Follow this link [duckduckgo.com](https://duckduckgo.com/)
```

2. Save
3. Click the link.

### Describe your Expected behavior

I expect, that external links are opened in my browser. Internal links, like a link to another room, should open within RocketChat.

### Are you willing to submit a code contribution?

- [ ] Yes, I am willing to submit a Pull Request!

**Comment by jeanfbrito** [comment]
Hello, this was fixed on the server update. Please update your server.
Even in browser it must be opening on the same tab and not in another one.

**Comment by stopal** [comment]
Hi @jeanfbrito ,

could you please post the corresponding release of the server version containing that fix? I couldn't find it in the release page

**Comment by andreasgrill** [comment]
@jeanfbrito 
I have the same issue with the current RocketChat version 5.4.2.

**Comment by andreasgrill** [comment]
@stopal 
I think I found the PR that fixes the issue on the server: https://github.com/RocketChat/Rocket.Chat/pull/27554
The PR was merged to the develop branch but was not included for the 5.x versions yet.

As a workaround for our RocketChat 5.4.2 instance I've used a custom script with the code from the PR that seems to work well enough until we update RocketChat to a version that contains the fix. 
I've added the following script to Layout -> Custom Script for logged in Users:

```
// Fixes an issue with opening topic links in the same tab
// This custom script should be removed when the RocketChat version contains a fix for #27554.
// See https://github.com/RocketChat/Rocket.Chat/pull/27554
const dompurify = require('dompurify');

// Add a hook to make all links open a new window
dompurify.addHook('afterSanitizeAttributes', (node) => {
	// set all elements owning target to target=_blank
	if ('target' in node) {
		node.setAttribute('target', '_blank');
		node.setAttribute('rel', 'nofollow noopener noreferrer');
	}
});
```
